exasol_transformers_extension/utils/huggingface_hub_bucketfs_model_transfer_sp.py

```diff
@@ -9,6 +9,13 @@
 from exasol_transformers_extension.utils.temporary_directory_factory import TemporaryDirectoryFactory
 
 
+def make_parameters_of_model_contiguous_tensors(model):
```

Comment:
i think it would be cleaner to have this in another file like "utils/model_operations"  or something.